docs/recipes/endpoint-testing-with-mongoose.md

```diff
@@ -6,9 +6,9 @@ This recipe shows you how to test your endpoints with AVA and Mongoose, assuming
 
 This recipe uses the following libraries:
 
-1. [MongoDB memory server](https://github.com/nodkz/mongodb-memory-server) (A MongoDB in-memory Server)
-2. [Supertest](https://github.com/visionmedia/supertest) (An endpoint testing library)
-3. [Mongoose](http://mongoosejs.com)
+1. [`mongod-memory-server`](https://github.com/nodkz/mongodb-memory-server) (A MongoDB in-memory Server)
+2. [SuperTest](https://github.com/visionmedia/supertest) (An endpoint testing library)
+3. [Mongoose ODM](http://mongoosejs.com)
```

Comment:
Why was this changed when "Mongoose" is know as "Mongoose" not "Mongoose ODM"?

Reply:
Ah I took that from the document title, but they don't refer to it that way elsewhere. Will change back, thanks!